airbyte-cdk/python/airbyte_cdk/sources/declarative/parsers/model_to_component_factory.py

```diff
@@ -829,7 +833,7 @@ def create_http_requester(self, model: HttpRequesterModel, decoder: Decoder, con
             disable_retries=self._disable_retries,
             parameters=model.parameters or {},
             message_repository=self._message_repository,
-            use_cache=model.use_cache,
+            use_cache=use_cache or False,
```

Comment:
Or False doesn’t seem right

Reply:
my bad🤦‍♂️

just quick fix to pass mypy 

moved `use_cache` variable definition under none value validation for `model.use_cache`